tests/statistics_test.sh

```diff
@@ -194,11 +196,39 @@ function month_statistics_Test
 function year_statistics_Test
 {
   local target_year="2019"
+  local year_data
   local msg="Currently, kw does not have any data for the requested year."
+  local line
 
-  ID=1
+  declare -a expected_cmd=(
+    "2020 summary"
+    "Total   Max  Min       Average"
+    "Build   19   00:03:40  00:00:06  00:00:23"
+    "Deploy  8    00:01:13  00:00:31  00:01:04"
+  )
+
+  setUp
```

Comment:
The setUp function is automatically invoked before each test; for this reason, you don't need to call it explicitly (in general).